core/src/main/resources/db/changelog/changesets/2024_08_28_in_person_kits.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: "in_person_kits"
+      author: mbemis
+      changes:
+        - addColumn:
+            tableName: kit_request
+            columns:
+              - column: { name: kit_origin_type, type: text, defaultValue: 'SHIPPED', constraints: { nullable: false } }
```

Comment:
maybe `distribution_method`?  origin type sounds more like it would be the provider of the kit (e.g GP or some 3rd party)